css/properties.json

```
@@ -4220,6 +4220,50 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/contain"
   },
+  "contain-intrinsic-size": {
+    "syntax": "[ none | <length> | auto <length> ]{1,2}",
+    "media": "visual",
+    "inherited": false,
+    "animationType": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "percentages": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "groups": [
+      "CSS Containment"
+    ],
+    "initial": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "appliesto": "allElements",
+    "computed": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "order": "perGrammar",
+    "status": "experimental",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/contain-intrinsic-size"
+  },
+  "contain-intrinsic-width": {
+    "syntax": "none | <length> | auto <length>",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "byComputedValueType",
+    "percentages": "no",
+    "groups": [
+      "CSS Containment"
+    ],
+    "initial": "none",
+    "appliesto": "elementsForWhichSizeContainmentCanApply",
+    "computed": "asSpecified",
```

Is this correct? Spec says "as specified, with [`<length>`](https://w3c.github.io/csswg-drafts/css-values-4/#length-value) values computed". 

The values for length (or in this case width are always specified). However if you were to use the `auto <length>` value it might be determined from a stored value.

---

I don't know. Maybe you need a new `asSpecifiedWithLengthValuesComputed` enum?

---

I think you're right - add the value in #606 
```suggestion
    "computed": "asSpecifiedWithLengthValuesComputed",
```